rules/functions

```diff
@@ -153,3 +153,7 @@ if [ -n "$(1)" ]; then \
     done; \
 fi
 endef
+
+TO_LOWER = $(shell echo $(1) | tr '[:upper:]' '[:lower:]')
```

Comment:
it is good you introduce this function, but I do not see you use it anywhere.

Reply:
Its is just a utility macro, it will be used later.